### ISSUE TITLE
[新功能] 自MLab HomePod 2.0 pro以来pytorch的更新

### ISSUE THREAD

**Issue by gemfield** [DeepVAC/TorchPod#48]
- 0dc40474fe      Peter Bell      Tue Jul 6 19:05:39 2021 -0700   Migrate glu from the THC to ATen (CUDA) (#61153);备注：glu是GatedLinearUnit；
- a69e947ffd      Freey0  Wed Jul 7 07:42:49 2021 -0700   avg_pool3d_backward: Port to structured (#59084)
- 45ce26c397      Xue Haotian     Wed Jul 7 12:32:43 2021 -0700   Port `isposinf` & `isneginf` kernel to structured kernels (#60633)
- baa518e2f6      Akshit Khurana  Wed Jul 7 12:37:51 2021 -0700   Add Int32 support for NNAPI (#59365)
- cf285d8eea      Akshit Khurana  Wed Jul 7 12:37:51 2021 -0700   Add aten::slice NNAPI converter (#59364)
- d26372794a      Akshit Khurana  Wed Jul 7 12:37:51 2021 -0700   Add aten::detach NNAPI converter (#58543)
- 0be228dd5f      Akshit Khurana  Wed Jul 7 12:37:51 2021 -0700   Add aten::flatten NNAPI converter (#60885)
- b297f65b66      Akshit Khurana  Wed Jul 7 12:37:51 2021 -0700   Add aten::div NNAPI converter (#58541)
- eab18a9a40      Akshit Khurana  Wed Jul 7 12:37:51 2021 -0700   Add aten::to NNAPI converter (#58540)
- 14d604a13e      Akshit Khurana  Wed Jul 7 12:37:51 2021 -0700   Add aten::softmax NNAPI converter (#58539)
- 179b3ab88c      Xiao Wang       Wed Jul 7 20:45:42 2021 -0700   [cuDNN] Enable cudnn_batchnorm_spatial_persistent for BatchNorm3d channels_last_3d (#59129)
